data/data-pipeline/data_pipeline/tests/sources/national_risk_index/test_etl.py

```diff
@@ -26,73 +96,56 @@ def test_init(self, mock_etl, mock_paths):
         etl = NationalRiskIndexETL()
         data_path, tmp_path = mock_paths
         input_csv = tmp_path / "NRI_Table_CensusTracts.csv"
-        output_dir = data_path / "dataset" / "national_risk_index_2020"
-        print(input_csv)
+
         # validation
-        assert etl.DATA_PATH == data_path
-        assert etl.TMP_PATH == tmp_path
         assert etl.INPUT_CSV == input_csv
-        assert etl.OUTPUT_DIR == output_dir
         assert etl.GEOID_FIELD_NAME == "GEOID10"
         assert etl.GEOID_TRACT_FIELD_NAME == "GEOID10_TRACT"
-
-    def test_transform(self, mock_etl):
-        """Tests the transform() method for NationalRiskIndexETL
-
-        Validates the following conditions:
-        - The columns have been renamed correctly
-        - The values for each tract has been applied to each of the block
-          groups in that tract
-        """
-        # setup - copy sample data into tmp_dir
+        assert etl.NAME == "national_risk_index"
+        assert etl.LAST_UPDATED_YEAR == 2020
+        assert (
+            etl.SOURCE_URL
+            == "https://hazards.fema.gov/nri/Content/StaticDocuments/DataDownload//NRI_Table_CensusTracts/NRI_Table_CensusTracts.zip"
+        )
+        assert etl.GEO_LEVEL == ValidGeoLevel.CENSUS_TRACT
+        assert etl.COLUMNS_TO_KEEP == [
+            etl.GEOID_TRACT_FIELD_NAME,
+            etl.RISK_INDEX_EXPECTED_ANNUAL_LOSS_SCORE_FIELD_NAME,
+            etl.EXPECTED_POPULATION_LOSS_RATE_FIELD_NAME,
+            etl.EXPECTED_AGRICULTURE_LOSS_RATE_FIELD_NAME,
+            etl.EXPECTED_BUILDING_LOSS_RATE_FIELD_NAME,
+        ]
```

Comment:
I am a little confused here from the perspective of updating / changing data -- like it would be really nice imo if all of this were housed in a single place outside of the test, with the idea that it could be shared by the etls themselves and also the tests -- but it's entirely possible i'm not understanding something here :)

Reply:
Oh I see -- your idea would be like, maybe there's a configuration file or something where we specify this, and then we're loading that configuration from both the ETLs and the tests? 

That is fairly reasonable to me (and not far off from the ideas of the Big Refactor). At the same time, with unit tests, it's really good to actually avoid being DRY (see this [post](https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html), or the principle of having tests be ["WET"](https://www.heroku.com/podcasts/codeish/102-whether-to-repeat-yourself-dry-damp-or-wet) (write everything twice). 

So I appreciate that if someone needs to update the columns to keep, that the test will explicitly fail and they have to manually update it. This is can catch an accidental modification of COLUMNS_TO_KEEP that would get loaded into the test from the same config and pass. 

What do you think?

Reply:
That seems reasonable. It's slightly at odds with the "update all pickles!" flag but :shrug: probably in a good way. I'm convinced!